<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="YP1000555LT" name="com.yash.RMS.Testcase.Login.TestCase_Login" tests="1" failures="1" timestamp="16 Mar 2018 09:26:33 GMT" time="1.777" errors="0">
  <testcase name="login" time="1.777" classname="com.yash.RMS.Testcase.Login.TestCase_Login">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.yash.RMS.POM.login.PageObjects_Login.txt_loggedinuser(PageObjects_Login.java:16)
	at com.yash.RMS.TestAction.login.Action_ValidLogin.loginvaliddata(Action_ValidLogin.java:12)
	at com.yash.RMS.Testcase.Login.TestCase_Login.login(TestCase_Login.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at com.yash.RMS.CustomProperties.XML.XMLRead(XML.java:545)
	at com.yash.RMS.CustomProperties.Main.main(Main.java:13)
]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- com.yash.RMS.Testcase.Login.TestCase_Login -->
